comp/core/config/config.go

```diff
@@ -92,6 +98,22 @@ func newConfig(deps dependencies) (Component, error) {
 		}
 	}
 
+	configRefreshInterval := config.GetDuration("agent_ipc.config_refresh_interval")
```

Comment:
We're most likely going to set a number of second in the configuration, no ? so maybe `config.GetDuration("agent_ipc.config_refresh_interval") * time.Second` ?

Reply:
I did it that way to be able to use less than 1s in the test, but you are right that it would make the config inconvenient, I'll edit it to avoid that